### ISSUE TITLE
Invalid protocol: 127.0.0.1

### ISSUE THREAD

**Issue by jjeejj** [Huachao/vscode-restclient#241]
- VSCode Version: 1.25.1
- OS Version: MacOS High Sierra 10.13.6 
- REST Client Version: 0.19.1

Steps to Reproduce:

selected a request api ，and   Send Request   then the error will show
`GET http://ip.taobao.com/service/getIpInfo.php?ip=myip`

![image](https://user-images.githubusercontent.com/15176971/44571275-e69f0a80-a7b2-11e8-9472-66dcaf5bd644.png)

this why?

**Comment by Huachao** [comment]
@jjeejj I can't repro it in my local machine. What's your proxy setting?

**Comment by Huachao** [comment]
@jjeejj I guess that your proxy is like `127.0.0.1:{port-number}/xxx`. And in this case, since you have a colon in the proxy, when the nodejs `URL` module parses the url, it thought the string before the first colon is the protocol name. I think the way to fix it is that, you just adjust the proxy to `http://127.0.0.1:{port-number}/xxx`.

**Comment by jjeejj** [comment]
yes , you are right .  set proxy ago.  but why  'when the nodejs URL module parses the url, it treats the text until the first colon is the protocol name'  ? 

The protocol schemes considered to be special by the [WHATWG URL](https://url.spec.whatwg.org/) Standard include:** ftp, file, gopher, http, https, ws, and wss **
